features/steps/insights_content_service.py

```diff
@@ -11,7 +11,7 @@ def check_build_commit(context):
 
 @then('BuildTime is a proper date')
 def check_build_time(context):
-    pattern = re.compile(r'.{3} .{3}  [0-9]* [0-9]{2}:[0-9]{2}:[0-9]{2} [AP]M [A-Z]{3} [0-9]{4}')
+    pattern = re.compile(r'.{3} .{3}[ ]{1,2}[0-9]* [0-9]{2}:[0-9]{2}:[0-9]{2} [AP]M [A-Z]{3} [0-9]{4}')
```

Comment:
are these changes part of this PR or leftovers? as they are not related to sha extractor

Reply:
correct, this is not related to the sha-extractor